language-bindings/go/go.mod

```diff
@@ -1,4 +1,4 @@
-module gitlab.alipay-inc.com/TNT_Runtime/ant-runtime/bindings/go
+module github.com/bytecodealliance/wasm-micro-runtime/language-bindings/go
```

Comment:
when using the Go bindings in a separate project, the module name must match the project URL, and the test below must use the same URL, too

Reply:
Should we also we put header and libraries files under `github.com/bytecodealliance/wasm-micro-runtime/language-bindings/go/packaged`?

Reply:
The header files and the host-native library are installed under `packaged/` by `language-bindings/go/build.sh`. But there's also the script `language-bindings/go/samples/build.sh`, which does practically the same but always puts the host-native library into `packaged/lib/amd64`, which is incorrect, e.g., on arm64. What about removing the latter, and maybe copying the command to build the test from the former to the latter?

Reply:
How about not removing `go/samples/build.sh`, instead, just letting it call the `go/build.sh` first and then building the sample, for example:
```bash
#!/usr/bin/env bash

# Copyright (C) 2019 Intel Corporation.  All rights reserved.
# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception

CUR_DIR=$PWD

pushd ${CUR_DIR}/.. > /dev/null 2>&1
./build.sh
popd > /dev/null 2>& 1

cd ${CUR_DIR}
rm -f test
go build test.go
./test
```

Reply:
See https://github.com/bytecodealliance/wasm-micro-runtime/pull/2681